takwimu/templates/takwimu/_includes/navbar/search.html

```diff
@@ -0,0 +1,40 @@
+<button type="button" class="close btn-close position-absolute" style="top: 1em; right: 1em;" data-dismiss="popover" aria-label="Close">
+    <span aria-hidden="true">&times;</span>
+</button>
+<div class="dropdown-menu-lg text-light container-fluid" style="padding: 10rem 0;">
+    <form action="{% url 'search' %}" method="get">
+        <div class="input-group col-6 mx-auto">
+            <input type="text" class="form-control border-0 py-3 px-5 text-center"  name="q" value="{{ query }}" id="search-takwimu" placeholder="Search Takwimu">
+
+            <a href="{% url 'search' %}" class="btn btn-search">
+                <i class="fa fa-search"></i>
+            </a>
+        </div>
+    </form>
+</div>
+<style>
+
+  .popover {
+      margin-top: -0.25rem;
+      left: 0;
+      right: -5px;
+      border-radius: 0;
+      max-width: 100vw;
+      background-color: #074c9c;
+    }
+
+    .bs-popover-auto[x-placement^=bottom] .arrow::after, 
+    .bs-popover-auto[x-placement^=bottom] .arrow::before, 
+    .bs-popover-bottom .arrow::after, 
+    .bs-popover-bottom .arrow::before {
+        border-bottom: 10px solid #074c9c;
+        border-left: 10px solid transparent;
+        border-right: 10px solid transparent;
+    }
+
+    .bs-popover-auto[x-placement^=bottom] .arrow::after,
+    .bs-popover-bottom .arrow::after  {
+        border-bottom-color: #074c9c;
+    }
```

Comment:
Are these popover classes used anywhere else?

Reply:
@DavidLemayian 
Yes, currently there is no easy was to style the popover arrow, it will improve as we go
I tried attaching a class to the popover with no luck (it was consuming much time).

Reply:
@mtuchi And this will therefore conflict with the rest of the uses. Please fix.

Reply:
@mtuchi doesn't work that way unfortunately right now. All styles are currently global in application but local to the page/route loaded.

Reply:
@DavidLemayian 
Okay let me see what i can do to improve this, will ping you once it's done